CITATION.cff

```diff
@@ -48,7 +48,9 @@ authors:
   given-names: Merel
 - family-names: Zabłocki
   given-names: Marcin
+- family-names: Khaustova
+  given-names: Elena
```

Comment:
This list is actually the same as our TSC list: https://github.com/kedro-org/kedro/pull/3565 (or it should be). We still need to cast a vote to add you to the TSC, which I think @astrojuanlu is doing soon 🤞 

That said, this does mean Jo Stichbury https://github.com/kedro-org/kedro/pull/3647 can be removed from this list now and Simon Brugman should be added https://github.com/kedro-org/kedro/pull/3780.